.buildkite/postgres-config.yaml

```diff
@@ -1,7 +1,7 @@
 # Configuration file used for testing the 'synapse_port_db' script.
 # Tells the script to connect to the postgresql database that will be available in the
 # CI's Docker setup at the point where this file is considered.
-server_name: "test"
+server_name: "localhost:8800"
```

Comment:
Is this a thing that should be here?

Reply:
Yup. The users in the test db are from `localhost:8800` and we now run the DB checks during the port DB run as they've been moved from synapse/app to the data stores.